tests/integration/_common.py

```diff
@@ -213,16 +213,20 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         if self._server.has_session(self._session_name):
             self._session.kill_session()
 
-    def interaction(self, value):
+    def interaction(self, value, wait_on_help=True, wait_on_playbook_status=None):
         """interact with the tmux session"""
         self._pane.send_keys(value, suppress_history=False)
-        help_not_on_screen = True
-        while help_not_on_screen:
+        ok_to_return = [False]
+        while not all(ok_to_return):
+            ok_to_return = []
             showing = []
             while not showing:
                 time.sleep(0.1)
                 showing = self._pane.capture_pane()
-            help_not_on_screen = not any(":help help" in line for line in showing)
+            if wait_on_help:
+                ok_to_return.append(any(":help help" in line for line in showing))
+            if wait_on_playbook_status is not None:
```

Comment:
Why not `if wait_on_playbook:` and make it a bool above? Otherwise, with this strict check, if someone calls with `wait_on_playbook_status=False`, we'll still wait on it right?

Reply:
my thinking here is that you may want to wait on a specific status, for instance, you may write a playbook that is known to fail, and then you may want to wait on FAILED

Reply:
ok, I get it :) thx, will fix

Reply:
Ah I see. I read it too quickly. Alright, I'm fine with either default then. It might still be worth making the check non strict just to save someone time in the future, but not a big deal.